Jenkinsfile

```diff
@@ -0,0 +1,13 @@
+@Library('github.com/fabric8io/fabric8-pipeline-library@master')
+def dummy
+goTemplate{
+  dockerNode{
+      goMake{
```

Comment:
Also what this would do is run your CI pipelines for pull requests, it will create a snapshot image and add a comment to the PR once it's available for testing.  Like this https://github.com/fabric8io/fabric8-init-tenant/pull/125#issuecomment-312240438